controllers/release/adapter.go

```diff
@@ -355,9 +373,31 @@ func (a *Adapter) finalizeRelease() error {
 	return nil
 }
 
-// registerGitOpsDeploymentStatus updates the status of the Release being processed by monitoring the status of the
+// registerDeploymentData adds all the Release information to its Status and marks it as processing.
```

Comment:
just like in a similar function (that you fixed the comment), it isn't all the Release information, just the deployment information

Reply:
Fixed.